incuna-sass/mixins/_em-scale.sass

```diff
@@ -23,30 +26,14 @@
 
 // You can also use variables to reference media query variables in your
 // project:
-//     @include em-scale(14px, $px768, 16px, $px1025, 20px)
+//     @include em-scale(14px, (($px768, 16px), ($px1025, 20px)))
 // or
-//     @include em-scale(14px, $tablet, 16px, $desktop, 20px)
+//     @include em-scale(14px, (($tablet, 16px), ($desktop, 20px)))
 
-
-@mixin em-scale($font-size-1, $breakpoint-1, $font-size-2, $breakpoint-2: false, $font-size-3: false, $breakpoint-3: false, $font-size-4: false, $breakpoint-4: false, $font-size-5: false, $breakpoint-5: false, $font-size-6: false)
+@mixin em-scale($font-base, $breakpoints-sizes-list: ())
```

Comment:
`: ()` defines the argument as optional, without which it becomes a required argument, so you must call it with a list.

Talking about it now, making the argument required would be a good thing I think, because there's no point calling it without a list of breakpoints (kind of already explained in the comments)